sls/resources/logback.xml

```diff
@@ -0,0 +1,37 @@
+<configuration>
+
+  <!-- Appender for dotty.tools.pc -->
+  <appender name="PC_LOG" class="ch.qos.logback.core.FileAppender">
+    <file>./.logs/pc.log</file>
+    <append>false</append>
+    <encoder>
+      <pattern>%date %-5level [%thread] %logger{36} - %msg%n</pattern>
+    </encoder>
+  </appender>
+
+  <!-- Appender for org.scala.abusers.pc -->
+  <appender name="SLS_LOG" class="ch.qos.logback.core.FileAppender">
+    <file>./.logs/sls.log</file>
+    <append>false</append>
+    <encoder>
+      <pattern>%date %-5level [%thread] %logger{36} - %msg%n</pattern>
+    </encoder>
+  </appender>
+
+  <!-- Logger for dotty.tools.pc -->
+  <logger name="dotty.tools.pc" level="DEBUG" additivity="false">
+    <appender-ref ref="PC_LOG"/>
+  </logger>
+
+  <!-- Logger for org.scala.abusers.pc -->
+  <logger name="org.scala.abusers" level="DEBUG" additivity="false">
+    <appender-ref ref="SLS_LOG"/>
+  </logger>
+
+  <!-- Root logger (optional) -->
+  <root level="INFO">
+    <!-- Add a console appender if desired -->
+    <!-- <appender-ref ref="STDOUT"/> -->
```

Comment:
we cannot have this, right? I would remove this lines then. However, we could log our stuff into stderr, right?